frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -210,6 +210,10 @@ action.GitMachete.FastForwardMergeSelectedToParentAction.description=Fast-forwar
 action.GitMachete.SyncSelectedToParentByMergeAction.text=Git Machete: Checkout and Sync to Parent by Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.GitMacheteContextMenu.text=Checkout and Sync to Parent by _Merge
 action.GitMachete.SyncSelectedToParentByMergeAction.description=Checkout this branch and merge parent into it
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.title=Merge operation warning
+action.GitMachete.SyncSelectedToParentByMergeAction.warning-dialog.message=Merge operation warning
```

Comment:
Whoops this needs to be refined ;D See https://github.com/VirtusLab/git-machete/blob/master/README.md#can-i-use-git-merge-for-dealing-with-stacked-prs